packages/backend/src/api/endpoints/UserApi.ts

```diff
@@ -166,4 +171,15 @@ export class UserAPI extends AbstractAPI {
       queryParams: { limit, offset },
     });
   }
+
+  public async verifyPassword(params: VerifyPasswordParams) {
+    const { userId, password } = params;
+    this.requireId(userId);
+
+    return this.request<{verified: true}>({
+      method: 'POST',
+      path: joinPaths(basePath, userId, 'verify_password'),
+      bodyParams: { userId, password },
```

Comment:
🔧 based on the [docs](https://clerk.dev/docs/reference/backend-api/tag/Users#operation/VerifyPassword) the userId is not part of the request `body` but is used in the path.